usr/share/rear/lib/layout-functions.sh

```diff
@@ -590,3 +587,27 @@ is_disk_a_pv() {
 function is_multipath_path {
     [ "$1" ] && type multipath &>/dev/null && multipath -c /dev/$1 &>/dev/null
 }
+
+# retry_command () is binded with REAR_SLEEP_DELAY and REAR_MAX_RETRIES.
+# This function will do maximum of REAR_MAX_RETRIES command execution
+# and will sleep REAR_SLEEP_DELAY after each unsuccessful command execution.
+# Function returns command result as soon as it succeeded.
+function retry_command () {
+    command="$1"
+
+    for pass in trial verification ; do
```

Comment:
If trial and verification were variables
the code would have to be
<pre>
    for pass in $trial $verification ; do
</pre>
;-)